lib/logstash/inputs/dead_letter_queue.rb

```diff
@@ -53,10 +58,18 @@ def register
     dlq_path = java.nio.file.Paths.get(File.join(@path, @pipeline_id))
     sincedb_path = @sincedb_path ? java.nio.file.Paths.get(@sincedb_path) : nil
     start_timestamp = @start_timestamp ? org.logstash.Timestamp.new(@start_timestamp) : nil
-    @inner_plugin = org.logstash.input.DeadLetterQueueInputPlugin.new(dlq_path, @commit_offsets, sincedb_path, start_timestamp)
+    @logstash_version = Gem::Version.new(LOGSTASH_CORE_VERSION)
```

Comment:
Nit: Does this need to be an instance variable?

Reply:
Fixed by 2a7e21eae2a56b705cc696436c7285bb119f59b3